tests/test_e2e_10_mef_eline.py

```diff
@@ -613,3 +696,538 @@ def test_080_create_and_remove_ten_circuits_ten_times(self):
+        data = response.json()
+        assert data['end_date'] == end_date.strftime(TIME_FMT)
+
+        # waiting to reach the new end timing
+        time.sleep(end_delay*60+5)
+
+        # Verify if the circuit is active
+        api_url = KYTOS_API + '/mef_eline/v2/evc/' + evc1
+        response = requests.get(api_url)
+        data = response.json()
+        assert data["active"] is False
+
+    def test_110_patch_bandwidth(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        bandwidth = 40
+        payload = {
+            "bandwidth": bandwidth
+        }
+
+        # It sets a new circuit's bandwidth
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['bandwidth'] == bandwidth
+
+        requests.delete(api_url + evc1)
+
+    def test_115_patch_priority(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        priority = 100
+        payload = {
+            "priority": priority
+        }
+
+        # It sets a new circuit's priority
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['priority'] == priority
+
+        s1, s2 = self.net.net.get('s1', 's2')
+        flows_s1 = s1.dpctl('dump-flows')
+        flows_s2 = s2.dpctl('dump-flows')
+        assert 'priority=100' in flows_s1
+        assert 'priority=100' in flows_s2
+
+    """It does not contain the queue information in the flow description"""
+    @pytest.mark.xfail
+    def test_120_patch_queue_id(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        queue_id = 3
+        payload = {
+            "queue_id": queue_id
+        }
+
+        # It sets a new circuit's queue_id
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['queue_id'] == queue_id
+
+        s1, s2 = self.net.net.get('s1', 's2')
+        flows_s1 = s1.dpctl('dump-flows')
+        flows_s2 = s2.dpctl('dump-flows')
+
+        assert 'set_queue:3' in flows_s1
+        assert 'set_queue:3' in flows_s2
+
+    def test_125_patch_dynamic_backup_path(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        dynamic_backup_path = True
```

Comment:
dynamic_backup_path is already True on create_evc(). Then you cannot be sure the PATCH was successful because it was already True or because it was changed to True. My suggestion:

```suggestion
        dynamic_backup_path = False
```

Reply:
Resolved with commit [98e5a1d](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/98e5a1d51ead09cc39153c17b6f0792d2f72733c)